client-libraries/0.1/how-to-guides/python/generate-an-address.md

```diff
@@ -0,0 +1,94 @@
+# Generate an address in Python
+
+**In this guide, you learn how to generate a new address for a [seed](root://getting-started/0.1/clients/seeds.md) with a given [security level](root://getting-started/0.1/clients/security-levels.md).**
+
+## Packages
+
+To complete this guide, you need to install the following package (if you're using Go modules, you just need to reference this package):
+
+```bash
+pip install pyota
+```
+
+## IOTA network
+
+In this guide, we connect to a [node](root://getting-started/0.1/network/nodes.md) on the [Devnet](root://getting-started/0.1/network/iota-networks.md#devnet).
+
+## Code walkthrough
+
+1. Import the packages
+
+    ```python
+    from iota import Iota
+    ```
+
+2. Define a seed for which to generate an address
+
+    ```python
+    seed = 'PUETPSEITFEVEWCWBTSIZM9NKRGJEIMXTULBACGFRQK9IMGICLBKW9TTEVSDQMGWKBXPVCBMMCXWMNPDX'
+    ```
+
+3. Connect to a node
+
+    ```python
+    api = Iota('https://nodes.devnet.iota.org:443', seed, testnet = True)
+    ```
+
+    :::info:
+    The `testnet` argument sets the [minimum weight magnitude](root://getting-started/0.1/network/minimum-weight-magnitude.md) to 9.
+    :::
+
+4. Define the security level that you want to use for your address
+
+    ```python
+    security_level = 2
+    ```
+
+5. Use the [`GetNewAddress()`](https://github.com/iotaledger/iota.go/blob/master/.docs/iota.go/reference/api_get_new_address.md) method to generate an unspent address. If the connected node has an input transaction that withdraws from the address with the given index, the node knows that the address is spent, so the library returns the next unspent address.
```

Comment:
in its current form it is not true for the python lib until we fix the bug I talked about